contracts/src/v0.8/llo-feeds/dev/FeeManager.sol

```diff
@@ -136,83 +134,58 @@ contract FeeManager is IFeeManager, ConfirmedOwner, TypeAndVersionInterface {
 
   /// @inheritdoc IERC165
   function supportsInterface(bytes4 interfaceId) external pure override returns (bool) {
-    return interfaceId == this.processFee.selector;
+    return interfaceId == this.processFee.selector || interfaceId == this.processFeeBulk.selector;
```

Comment:
I wonder if we should actually use this method when setting the contract addresses elsewhere, since currently only the version on the `Verifier` contract is used.

Reply:
Good idea, maybe when we link the contracts together. I'll create a ticket.